### PR TITLE
feat: document list of allowed remap changes (GSF-5845)

### PR DESCRIPTION
**Pull request by harryjhornby**
Thank you for contributing to the documentation.

Your Jira ticket is:
[GSF-5845](https://genesisglobal.atlassian.net/browse/GSF-5845)

Have you provided changes for all the relevant versions: next, 2022.4, 2023.1, etc?
Yes

Have you checked all new or changed links?
Yes

Is there anything else you would like us to know?
No

For reference: 

- if you are an internal contributor:
  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 
- If you are an external contributor:
  - We have an [external contribution guide](../Type-of-contribution)

**This week's exciting excerpt from the style guide**
LISTS. Only use numbered lists when the sequence is important. Usually, that is a set of instructions.
For example:
1. Change the dictionary files.
2. Run genesisInstall.
3. Run remap.
In all other cases, use an **unnumbered** list. OK?

### CODE REVIEW

File: src/client-modules/docs.iife.min.js
Comment (harryjhornby) [inline src/client-modules/docs.iife.min.js:1]: Feels like this should be in a gitignore and not committed

**Comment by GuiWar** [comment]
Hi Harry. It seems that your PR is causing a conflict. Can you resolve it before I can resolve it before I can approve it, please?

**Review by wjhendry — APPROVED** [review]

**Comment by wjhendry** [comment]
I'll merge this if you are happy, @harryjhornby

**Review by wjhendry — APPROVED** [review]

**Pull request merged**
